python/pb_unit_tests/BUILD

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2009-2021, Google LLC
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#     * Redistributions of source code must retain the above copyright
+#       notice, this list of conditions and the following disclaimer.
+#     * Redistributions in binary form must reproduce the above copyright
+#       notice, this list of conditions and the following disclaimer in the
+#       documentation and/or other materials provided with the distribution.
+#     * Neither the name of Google LLC nor the
+#       names of its contributors may be used to endorse or promote products
+#       derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL Google LLC BE LIABLE FOR ANY
+# DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+TESTS = [
+    "descriptor_database_test",
+    "descriptor_pool_test",
+    "descriptor_test",
+    "generator_test",
+    "json_format_test",
+    "keywords_test",
+    "message_factory_test",
+    "message_test",
+    "proto_builder_test",
+    "reflection_test",
+    "service_reflection_test",
+    "symbol_database_test",
+    "text_encoding_test",
+    "text_format_test",
+    "unknown_fields_test",
+    "well_known_types_test",
+    "wire_format_test",
+]
+
+[
+  py_test(
+      name = test,
+      main = test + "_wrapper.py",
+      srcs = [
+          test + "_wrapper.py",
+      ],
+      deps = [
+          "//python:message_ext",
+          "@com_google_protobuf//:python_srcs",
+          "@com_google_protobuf//:python_common_test_protos",
+          "@com_google_protobuf//:python_specific_test_protos",
+      ],
+      legacy_create_init = False,
+  ) for test in TESTS
+]
```

Comment:
I usually prefer to stamp out the parts of a build file mutliple times so that tools like buildozer can modify it more easily.  historically speaking I have always been burned by build files that contain complex logic like this.

Reply:
The repetition hurts (I worry about the toil in making future manual changes), but I will try. Done.